content/foundations/physical_properties/physical_properties_conductivity.rst

```diff
@@ -31,7 +31,7 @@ Current density represents the amount of current flowing per unit cross-sectiona
 
 In essence, the electrical conductivity defines how easy it is for current to flow though a particular material when subjected to an electric field.
 In conductive materials (larger :math:`\sigma`), free charges move quite easily and strong currents may be induced by a relatively weak electrical field.
-In contrast, resistive materials (smaller :math:`\sigma`) require strong electrical fields in order to produce any significant current.
+In contrast, resistive materials (smaller :math:`\sigma`) require strong electrical fields or voltages in order to produce any significant current.
```

Comment:
here, I don't think "voltages" is the term to use (it is really potential differences) . I would suggest leaving it at strong electric fields (strong potential differences produce strong electric fields, so they are two sides of the same coin. Thoughts @dccowan, @sgkang, @dougoldenburg ?